ddtrace/opentracer/constants.py

```diff
@@ -0,0 +1,13 @@
+class ConfigKeys(object):
```

Comment:
Minor nit but it would be a bit more idiomatic to use a [`namedtuple`](https://docs.python.org/2.7/library/collections.html#collections.namedtuple) here instead of a class. It presents a bit less overhead and results in an immutable data structure while providing similar convenient access methods.

Reply:
Perfect, I kinda figured a regular old object wouldn't be ideal.